backend/src/database/capabilities/index.ts

```diff
@@ -61,7 +61,7 @@ const capabilities = [
   },
   {
     capability: "reports_read_all",
-    allowed: ["admin"],
+    allowed: ["admin", "manager", "gdx"],
```

Comment:
shouldn't you indicate in the changelog that you need to run the seed to update this information.